components/client/src/main/java/com/hotels/styx/client/OriginsInventory.java

```diff
@@ -22,13 +22,14 @@
 import com.google.common.eventbus.EventBus;
 import com.google.common.eventbus.Subscribe;
 import com.hotels.styx.api.Announcer;
+import com.hotels.styx.api.HttpHandler;
 import com.hotels.styx.api.Id;
+import com.hotels.styx.api.client.RemoteHost;
 import com.hotels.styx.api.client.ActiveOrigins;
 import com.hotels.styx.api.client.ConnectionPool;
 import com.hotels.styx.api.client.Origin;
-import com.hotels.styx.api.client.OriginsSnapshot;
 import com.hotels.styx.api.client.OriginsChangeListener;
-import com.hotels.styx.api.client.RemoteHost;
```

Comment:
This is really picky, but if you are going to make other changes, maybe  you can try re-organizing the imports so they are ordered in alphabetical order again?

Reply:
ok